addons/strings.yaml

```
@@ -92,6 +92,41 @@ commonStrings:
     value: An option, under “Preferences”, to enable support for RCS/MMS messaging, Apple features like CarPlay and AirDrop, as well as connections to local devices.
     comment: First bullet point with a specific update in 2.31 - shown only on iOS
 
+233updateMessage:
+  bullet1:
+    value: "[all] Finish transitioning subscription management to the Mozilla Accounts website."
```

```suggestion
    value: "[all] Finish transitioning subscription management to the Mozilla accounts website."
```

Did you mean "Finished"? "Finish" sounds like a plan, not a release note. Also, accounts always lowercase.